e2e/tests/free5gc/007-edge01-ueransim.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   name: edge01-ueransim
 spec:
   upstream:
-    repo: nephio-example-packages
+    repo: catalog-workloads-free5gc
```

Comment:
This change requires that the https://github.com/nephio-project/catalog/pull/22 to be merged